x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -253,9 +253,14 @@ export const sortByEnabledRules = () => {
   cy.get(SORT_RULES_BTN).contains('Enabled').click({ force: true });
 };
 
+export const waitForRulesTableToShow = () => {
+  // Wait up to 5 minutes for the rules to show up as in CI containers this can be very slow
```

Comment:
Is this something happening in the context of these tests or in general?

Reply:
@MadameSheema I experienced all sorts of timeouts in various parts of these tests yesterday and had to increase the timeout in several places. This felt like:

- a general issue with the speed of e2e tests (maybe they just got slower?)
- plus even more tangible issues with the speed of certain actions: e.g. installing an integration and loading the rules table